### ISSUE TITLE
GCP - Metadata Startup Script not running on F5 LTM

### ISSUE THREAD

**Issue by brstallings-clgx** [f5devcentral/f5-terraform#2]
I've been working on this for several days now. 
For some reason, when I use "https://github.com/f5devcentral/f5-terraform/tree/master/modules/providers/gce/infrastructure/proxy/standalone/1nic/byol" with bigip.tf and user_data.tpl, the startup-script never actually runs on the deployed F5 instance.
I've tried this using the Google compute module as well as the instance module.  
Once the server finishes building, I see the startup-script under metadata properties.  When I manually ssh to the server and check the /config folder, there is no cloud folder.  Also when I check the /var/log folder, there is no startup-script.log.
It's like the Google instance received the startup-script but never injected it into the F5 operating system in order to run it.
I've even removed all variables in the user_data.tpl file in order to simplify things, but nothing seems to work.

Basically, all I would like the tpl file to do is to set the initial admin password on the F5 and save the config.  From there my ansible setup will deploy the configuration.

[main.txt](https://github.com/f5devcentral/f5-terraform/files/4483912/main.txt)
[user_data.txt](https://github.com/f5devcentral/f5-terraform/files/4483914/user_data.txt)
